mods/parsoid.js

```diff
@@ -24,6 +24,25 @@ function ParsoidService(options) {
     options = options || {};
     this.parsoidHost = options.parsoidHost
         || 'http://parsoid-lb.eqiad.wikimedia.org';
+    // Set up operations
+    var self = this;
+    this.operations = {
+        getPageBundle: function(restbase, req) {
```

Comment:
I agree with you on not exposing `pagebundle`, it's used only internally in this module anyway.